x-pack/plugins/lists/server/routes/summary_exception_list_route.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { transformError } from '@kbn/securitysolution-es-utils';
+import {
+  SummaryExceptionListSchemaDecoded,
+  summaryExceptionListSchema,
+} from '@kbn/securitysolution-io-ts-list-types';
+import { EXCEPTION_LIST_URL } from '@kbn/securitysolution-list-constants';
+
+import type { ListsPluginRouter } from '../types';
+
+import {
+  buildRouteValidation,
+  buildSiemResponse,
+  getErrorMessageExceptionList,
+  getExceptionListClient,
+} from './utils';
+
+export const summaryExceptionListRoute = (router: ListsPluginRouter): void => {
+  router.get(
+    {
+      options: {
+        tags: ['access:lists-summary'],
+      },
+      path: `${EXCEPTION_LIST_URL}/_summary`,
```

Comment:
The reason we do the underscore such as in some cases `_import` is to mimic what elastic and Kibana does when it is action oriented. If you're just getting a summary of something and feel it is not an action that is being performed you can use just a normal `summary` without an underscore.

This looks fine with an underscore as is, though. It's kind of a grey area of when to use it vs. not use it in some cases. Just pointing out when/why I try to use it. I don't think there are hard/fast rules on it.

Reply:
That make sense to me, I'll change it, thanks! 👍